nano/styles/_misc.less

```diff
@@ -0,0 +1,127 @@
+#uiNoScript {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    margin: -60px 0 0 -150px;
+    width: 280px;
+    height: 120px;
+    background: #ffffff;
+    border: 2px solid #ff0000;
+    color: #000000;
+    font-size: 10px;
+    font-weight: bold;
+    z-index: 9999;
+    padding: 0px 10px;
+    text-align: center;
+}
+
+/**********************************
+     .notice | Caution Stripes
+**********************************/
+.notice {
+    position: relative;
+    background: url(uiNoticeBackground.jpg) 50% 50%;
+    color: #000000;
+    font-size: 12px;
+    font-style: italic;
+    font-weight: bold;
+    padding: 3px 4px 3px 4px;
+    margin: 4px 0 4px 0;
+}
+
+.noticePlaceholder {
+    position: relative;
+    font-size: 12px;
+    font-style: italic;
+    font-weight: bold;
+    padding: 3px 4px 3px 4px;
+    margin: 4px 0 4px 0;
+}
+
+.notice.icon {
+    padding: 2px 4px 0 20px;
+}
+
+.notice img {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 16px;
+    height: 16px;
+}
+
+div.notice {
+    clear: both;
+}
+/********************\
+\********************/
+
+
+/**********************************
+    PDA, DNA Modifier styles
+    Don't fit anywhere else.
```

Comment:
@Tastyfish Ah, yeah, I thought about it- buutt it would make the already-messy gulpfile a bit more messy. Eh. I guess I'll fix it later when I redo the CSS.